lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -18,3 +18,14 @@
 .nx-radio__input:checked + .nx-radio__circle .nx-radio__outer-circle {
   fill: none;
 }
+
+.nx-radio-checkbox__input:checked:disabled {
+  + .nx-radio-checkbox__control.nx-radio__circle {
```

Comment:
Adding `.nx-radio__circle` to this and the next one seems unneeded

Reply:
> Adding `.nx-radio__circle` to this and the next one seems unneeded

7b24206ff7820928b158a88caa71208fc898b12e